qubes/storage/__init__.py

```diff
@@ -158,6 +160,61 @@ def __xml__(self):
         config = _sanitize_config(self.config)
         return lxml.etree.Element('volume', **config)
 
+    @property
+    def ephemeral(self):
+        '''Should this volume be encrypted with an ephemeral key in dom0?
```

Comment:
Can use please replace triple single quotes by triple double quotes everywhere please? (possibly as a separate commit) We should add such checks into lint test.

Reply:
That would be a separate PR ― this is consistent with the rest of the code, IIUC.

Reply:
Separate commit on files you edited is fine. Generally this is what I do but as you prefer :)